packages/dom-utilities/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/dom-utilities
+
+**Dom Utilities components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
```

Comment:
is this still true?

Reply:
(no i don't believe this is true, good catch. this is the default commentary from the create-project template)